v1/src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -198,11 +189,7 @@ public interface SplunkOptions extends PipelineOptions {
         },
         description = "Source of the token passed. One of PLAINTEXT, KMS or SECRET_MANAGER.",
         helpText =
-            "Source of the token. One of PLAINTEXT, KMS or SECRET_MANAGER. This parameter "
-                + "must be provided if secret manager is used. If tokenSource is set to KMS, "
-                + "tokenKMSEncryptionKey and encrypted token must be provided. If tokenSource is set to "
-                + "SECRET_MANAGER, tokenSecretId must be provided. If tokenSource is set to PLAINTEXT, "
-                + "token must be provided.")
+            "Source of the token. One of `PLAINTEXT`, `KMS` or `SECRET_MANAGER`. This parameter must be provided if Secret Manager is used. If `tokenSource` is set to `KMS`, `tokenKMSEncryptionKey` and encrypted `token` `must` be provided. If `tokenSource` is set to `SECRET_MANAGER`, `tokenSecretId` `must` be provided. If `tokenSource` is set to `PLAINTEXT`, `token` `must` be provided.")
```

Comment:
```suggestion
            "The source of the token. The following values are allowed: `PLAINTEXT`, `KMS`, and `SECRET_MANAGER`. You must provide this parameter when Secret Manager is used. If `tokenSource` is set to `KMS`, `tokenKMSEncryptionKey`, and encrypted, then `token` must be provided. If `tokenSource` is set to `SECRET_MANAGER`, then `tokenSecretId` must be provided. If `tokenSource` is set to `PLAINTEXT`, then `token` must be provided.")
```

Reply:
I'm confused by this sentence: If `tokenSource` is set to `KMS`, `tokenKMSEncryptionKey` and encrypted, then `token` must be provided.

We should clarify exactly what this is saying, because I can't parse it.